.github/workflows/build-binaries.yml

```diff
@@ -0,0 +1,77 @@
+name: binaries
```

Comment:
Nit: Call it something like "nightly-binaries"?

Reply:
They aren't really nightlies, because they happen on every push, not every night. That said, I think I'm just going to file another follow up PR that standardizes our workflow file naming (consistent capitalization, etc.)

Reply:
Why though? Imo "nightly" does not need to imply the time of the release, e.g rustc nightlies or other ToT builds.

Reply:
But the rustc nightly channel (what people collectively know as "nightlies") is only updated [once a day](https://rust-lang.github.io/rustup-components-history/). The CI system may produce binaries for every commit, but those aren't "nightlies".

Reply:
ok, scrap the "nigthly" and just call it ToT binaries in the followup.

Reply:
FWIW I don’t think we’d be the first project to use “nightly” for things that aren’t technically nightly and I think it conveys the right general idea as a term of art. But if you can think of a better term then I’m all for that.